README.md

```diff
@@ -466,7 +466,7 @@ The code in this repository is licensed under `MIT` (see root `LICENSE`), except
 [image-icon]: https://raw.githubusercontent.com/eclipse-cdt-cloud/theia-trace-extension/master/doc/images/theia-trace-extension-icon.png
 [image-open-browser]: https://raw.githubusercontent.com/eclipse-cdt-cloud/theia-trace-extension/master/doc/images/theia-trace-extension-open-browser.png
 [image-open-view]: https://raw.githubusercontent.com/eclipse-cdt-cloud/theia-trace-extension/master/doc/images/theia-trace-extension-open-view.gif
-[image-open-with]: https://raw.githubusercontent.com/eclipse-cdt-cloud/theia-trace-extension/master/doc/images/theia-trace-extension-open-with-trace-viewer.gif
+[image-open-with-trace-viewer]: https://raw.githubusercontent.com/eclipse-cdt-cloud/theia-trace-extension/master/doc/images/theia-trace-extension-open-with-trace-viewer.gif
```

Comment:
Github caches images. So, even if you change the source file of the image it will still show the old file on the webpage. So, what I usually do is to have a version number on the file, include that new file and delete the old one with the old name (see below for an example).

Reply:
I've seen your comment below, about being ok with "open in" - I'll restore that version and it will fix this comment, since the .gif file is named differently.